src/dd2.jl

```diff
@@ -1,20 +1,17 @@
 ## Matlab Author: Rodrigo de Barros Vimieiro
 ## Translated by Sonia Minseo Kim
 
-## 2-D Branchless Distance Driven Code
```

Comment:
Why did you remove "Branchless" here?
There are two versions of DD: regular and branchless.
I think this code is the branchless version so we should say that, right?

Reply:
I thought it was technically not branchless because we have the if statements

Reply:
The inner-most loop has no `if` statements, and that is what matters for performance!

Reply:
I see! And on a side note, this version of the code works now. Is there anything else you think I need to change?